Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:alpine

```

Comment:
It would be nice to fix a version

Reply:
Updated node and postgres versions to match with used on heroku